tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -238,7 +238,7 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
-	k_sleep(K_MSEC(400));
+	k_sleep(K_MSEC(1000));
```

Comment:
Keep this 400 millisecond. (Sufficient to analyse periodic adv PDUs in imported Ellisys traces).

Reply:
Fixed.